Laravel.gitignore

```diff
@@ -1,5 +1,7 @@
 vendor/
 node_modules/
+.rocketeer/
```

Comment:
If these rules are related, could you move them out to their own section like this:

```
# Rocketeer - task runner and deployment package
.rocketeer/
.phpintel/
```